RocksDBStore/Store.cs

```diff
@@ -0,0 +1,100 @@
+using Neo.Persistence;
+using RocksDbSharp;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Neo.Plugins.Storage
+{
+    internal class Store : IStore
+    {
+        private static readonly byte[] SYS_Version = { 0xf0 };
+        private readonly RocksDb db;
+        private readonly Dictionary<byte, ColumnFamilyHandle> _families = new Dictionary<byte, ColumnFamilyHandle>();
+
+        public Store(string path)
+        {
+            var families = new ColumnFamilies();
+            for (int x = 0; x <= byte.MaxValue; x++)
+                families.Add(new ColumnFamilies.Descriptor(x.ToString(), new ColumnFamilyOptions()));
+            db = RocksDb.Open(Options.Default, path, families);
```

Comment:
According to the UT this spend a lot of time the first time (1 minute). After this, it will spend less, but also more than levelDB.

Reply:
Maybe we don't need to add the families because of:
https://github.com/neo-project/neo-plugins/blob/2c905bd789c33af64f547b2bf8f6ba3a7c2ce2d4/RocksDBStore/Options.cs#L15